chispa/dataframe_comparer.py

```diff
@@ -5,14 +5,22 @@
 import chispa.six as six
 from functools import reduce
 
+
 class DataFramesNotEqualError(Exception):
    """The DataFrames are not equal"""
    pass
 
 
-def assert_df_equality(df1, df2, ignore_nullable = False, transforms = [], allow_nan_equality = False):
-    df1 = reduce(lambda acc, fn: fn(acc), transforms, df1)
-    df2 = reduce(lambda acc, fn: fn(acc), transforms, df2)
+def assert_df_equality(df1, df2, ignore_nullable=False, transforms=None, allow_nan_equality=False, ignore_column_order=False, ignore_row_order=False):
+    if transforms is None:
+        transforms = []
+    if ignore_column_order:
+        transforms.append(lambda df: df.select(sorted(df.columns)))
+    if ignore_row_order:
+        transforms.append(lambda df: df.sort(df.columns))
+    if transforms is not None:
+        df1 = reduce(lambda acc, fn: fn(acc), transforms, df1)
+        df2 = reduce(lambda acc, fn: fn(acc), transforms, df2)
```

Comment:
Hmm, I would try to combine this into a single statement with the previous `if transforms is None`. Or, I would just remove the `if` statement here since it will always run.

Reply:
Good catch, thanks!  Fixed.